model.js

```diff
@@ -66,16 +66,39 @@ module.exports.fetchComByReviewId = (reviewId) => {
 }
 
 module.exports.addComment = (IdUserBody) => {
```

Comment:
I dont know why you have decided to pass these things in as an array, I would prefer to pass them in as an object and convert then to an array here for pg to work with